build_npm.ts

```diff
@@ -40,6 +40,11 @@ await build({
     bugs: {
       url: "https://github.com/slackapi/deno-slack-sdk/issues",
     },
+    // sets the minimum engine to node v14
```

Comment:
I'd add a note in this comment as to _why_ we set this minimum (because dnt transpilation-generated code seems to only be able to successfully compile as far back ES2020)

Reply:
Thanks! I've added it :)